arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -225,12 +225,12 @@
             "$ref": "#/parameters/RegistryNameParameter"
           },
           {
-            "name": "registryUpdateParameters",
+            "name": "registry",
             "in": "body",
             "description": "The parameters for updating a container registry.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/RegistryUpdateParameters"
+              "$ref": "#/definitions/Registry"
```

Comment:
Registry cannot be a part of the request since it has a required property (location), either revert to the old model or create a different model that contains exactly the properties that can be updated.